code/git.cheat

```diff
@@ -58,8 +58,11 @@ cd <toplevel_directory>; \
 # Stage all files in project
 git add -A
 
-# Saves the changes to a file in a commit
-git commit -m <message>
+# Create commit for staged files
```

Comment:
I also changed the cheat description because the original description only mentions committing changes for a single file, but the commit will apply to all staged files.